transaction_adaptor.py

```diff
@@ -0,0 +1,93 @@
+
+import csv
+import string
+import datetime
+
+def discover_adaptor(file_path : str):
+    '''
+    Discover
+    export as CSV file
+    Trans. Date, Post Date, Description, Amount, Category
+    '''
+    transaction_list = []
+    with open(file_path) as discover_csv:
+        discover_data = list(csv.DictReader(discover_csv))
+    
+    for row in discover_data:
+        transaction_list.append(make_transaction_from_discover(row))
+    return transaction_list
+
+def make_transaction_from_discover(discover_transaction):
+    date = change_date(discover_transaction["Trans. Date"])
+    amount = change_amount(discover_transaction["Amount"])
+    merchant = change_merchant(discover_transaction["Description"])
+    transaction_object = Transaction(date = date, amount = amount, merchant = merchant)
+    return transaction_object
+
+def change_date(trans_date : str):
+    correct_date = trans_date.split(sep = "/")
+    correct_date = list(map(int, correct_date))
+    correct_date = month_day_year_to_date(*correct_date)
+    return correct_date
+
+def month_day_year_to_date(month, day, year):
+    return date(year, month, day)
+
+def change_amount(amount : str):
+    correct_amount = float(amount)
+    return correct_amount
+
+def change_merchant(merchant : str):
```

Comment:
What are you doing this for?

Is it necessary?

If so, can you give it a descriptive name, and a docstring?